src/lib.rs

```diff
@@ -64,10 +64,10 @@ fn update() -> Result<(), Box<::std::error::Error>> {
 }
 ```
 
-Separate utilities are also exposed:
+Separate utilities are also exposed (**NOTE**: the following example _requires_ the `archive-tar` feature):
```

Comment:
does the github example now require you to specify features? (`cargo run --example github`)

Reply:
Yes.

```text
> cargo run --example github
[...]
The new release will be downloaded/extracted and the existing binary will be replaced.
Do you want to continue? [Y/n]
Downloading...
[00:00:00] [========================================] 1.44MB/1.44MB (0s) Done
Extracting archive... thread 'main' panicked at 'Archive extension 'tar' not supported, please enable 'archive-tar' feature!', src/lib.rs:277:21
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Running `cargo run --example github --features "archive-tar compression-flate2"` succeeds though. Fixed in 682b2f6b6935db048dd9381647094363.